README.md

```diff
@@ -15,7 +15,10 @@ GRADIENT_WORKSPACE_ID=...
 
 1. Clone this repo
 2. Run `poetry install`
-3. Run `poetry run fine_tune` to run fine-tuning example, or `poetry run embeddings` to run embeddings example
+3. Run any of the following:
+   - `poetry run fine_tune`
+   - `poetry run embeddings`
+   - `poetry run guidance`
```

Comment:
Wasn't an intended change, please verify if this is intended.

Reply:
yeah this was a change I made a little too late to go in the last update. It should be included.